tsl/test/expected/split_chunk.out

```diff
@@ -0,0 +1,421 @@
+-- Split at multiple points. Not supported yet.
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk', 'time', split_at => '{ 2024-01-04 10:00, 2024-01-07 12:00 }'::timestamptz[]);
+ERROR:  invalid type 'timestamp with time zone[]' for split_at argument
+-- Try to split something which is not a chunk
+call split_chunk('splitme', 'time');
+ERROR:  chunk not found
+-- Split a chunk with unsupported access method
+alter table _timescaledb_internal._hyper_1_1_chunk set access method testam;
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ERROR:  access method "testam" is not supported for split
+alter table _timescaledb_internal._hyper_1_1_chunk set access method heap;
+-- Split an OSM chunk is not supported
+reset role;
+update _timescaledb_catalog.chunk ch set osm_chunk = true where table_name = '_hyper_1_1_chunk';
+set role :ROLE_DEFAULT_PERM_USER;
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ERROR:  cannot split OSM chunks
+reset role;
+update _timescaledb_catalog.chunk ch set osm_chunk = false where table_name = '_hyper_1_1_chunk';
+set role :ROLE_DEFAULT_PERM_USER;
+-- Split a frozen chunk is not supported
+select _timescaledb_functions.freeze_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ freeze_chunk 
+--------------
+ t
+(1 row)
+
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ERROR:  cannot split frozen chunk "_timescaledb_internal._hyper_1_1_chunk" scheduled for tiering
+select _timescaledb_functions.unfreeze_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ unfreeze_chunk 
+----------------
+ t
+(1 row)
+
+-- Split a compressed/columnstore chunk is not supported
+begin;
+call convert_to_columnstore('_timescaledb_internal._hyper_1_1_chunk');
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ERROR:  splitting a compressed chunk is not supported
+rollback;
+-- Split by non-owner is not allowed
+set role :ROLE_1;
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk');
+ERROR:  must be owner of table _hyper_1_1_chunk
+set role :ROLE_DEFAULT_PERM_USER;
+\set ON_ERROR_STOP 1
+call split_chunk('_timescaledb_internal._hyper_1_1_chunk', split_at => '2024-01-04 00:00');
+select chunk_name, range_start, range_end
+from timescaledb_information.chunks
+order by chunk_name, range_start, range_end;
+    chunk_name    |         range_start          |          range_end           
+------------------+------------------------------+------------------------------
+ _hyper_1_1_chunk | Wed Jan 03 16:00:00 2024 PST | Thu Jan 04 00:00:00 2024 PST
+ _hyper_1_3_chunk | Thu Jan 04 00:00:00 2024 PST | Wed Jan 10 16:00:00 2024 PST
+(2 rows)
+
+select * from _timescaledb_catalog.dimension_slice
+order by id, range_start, range_end;
+ id | dimension_id |   range_start    |    range_end     
+----+--------------+------------------+------------------
+  2 |            1 | 1704326400000000 | 1704355200000000
+  3 |            1 | 1704355200000000 | 1704931200000000
```

Comment:
Nit. Can we `_timescaledb_functions.to_timestamp` here to make the output test easy for readers?